src/utils.cpp

```diff
@@ -1628,6 +1629,179 @@ int get_skill(struct char_data *ch, int skill, int &target)
     }
   }
 
+  // Reflex Recorder, adds +1 die to Physical Skill tests for its associated skill. -Vile
+  // Utilizes Value 6 to reference its associated skill.
+  if (recorder)
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_EDGED_WEAPONS && GET_OBJ_VAL(bio, 5) == 1) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_POLE_ARMS && GET_OBJ_VAL(bio, 5) == 2) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_WHIPS_FLAILS && GET_OBJ_VAL(bio, 5) == 3) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_CLUBS && GET_OBJ_VAL(bio, 5) == 4) {
```

Comment:
Wait, did you define a custom mapping for these skills? SKILL_CLUBS is 6, but you have 4 here. If you did, rip it out and start again, because this is going to be a nightmare to maintain (imagine glancing at this a year later and trying to intuitively understand what's going on).

Happy to help you figure out the logic for that in iedit.

Reply:
To clarify, the ideal state here is that value 5 would contain the skill that we're intending to have the reflex recorder apply to.